lib/pipeline.ts

```diff
@@ -160,7 +161,13 @@ export interface PipelineProps {
    *
    * @default - Duration.hours(8)
    */
-  buildTimeout?: Duration;
+  readonly buildTimeout?: Duration;
+
+  /**
+   * Slack channel configurations where failure notification
+   * in this pipeline should be sent.
+   */
+  readonly failureNotifySlack?: chatbot.SlackChannelConfiguration[];
```

Comment:
Would have preferred for failure notifications to have been an enum-like type, to pick between Slack and Chime .